voicer-client/src/components/voice/Voice.jsx

```diff
@@ -32,12 +34,13 @@ export default function Voice() {
           setNameMatchesDB(true);
         });
     } else {
-      //console.log("there is no param")
+      //parse search string into multiple search items
+      //console.log(`sending to: ${url}/api/users${searchTags}`)
       axios
-        .get(`${url}/api/users`)
+        .get(`${url}/api/users${searchTags}`)
```

Comment:
So the backend will receive something like "TAG1 TAG2 TAG3"?

Reply:
?tag0=this&tag1=is